rocAL_pybind/amd/rocal/types.py

```diff
@@ -58,6 +59,7 @@
 from rocal_pybind.types import RGB_PLANAR
 
 #     RocalTensorLayout
+from rocal_pybind.types import NONE
```

Comment:
Are we using NONE layout anywhere?
If not please don't add it as it was already a concern raised

Reply:
None is used as the default layout in case user doesn't specify a layout. It is used in cosmoflow where the labels shape is (4,) and doesn't belong to any layout